docs/learn-legitimate-projects.md

```diff
@@ -1,242 +1,162 @@
 ---
-id: learn-legitimate-projects
-title: How to Identify Legitimate Projects
-sidebar_label: How to Identify Legitimate Projects
+id: how-to-dyor
+title: How to Do Your Own Research (DYOR)
+sidebar_label: How to Do Your Own Research
 ---
 
-There are many projects that are being built, or intend to build, on both Polkadot and Kusama.
-However, there are also many that claim to do so, but actually have either no such intention, they
-don't have the resources to pull it through or they're outright trying to scam people by misusing
-the Polkadot and Kusama brands.
+Many projects are being built, or intend to build, on both Polkadot and Kusama. However, many claim to do so but have either no such intention, they don't have the resources to pull it through, or they're outright trying to scam people by misusing the Polkadot and Kusama brands.
```

Comment:
Perhaps more concise, something along the lines of: "...many claim to do so but have either no such intention or capacity, or they're outright trying to scam people by misusing the Polkadot and Kusama brands."

Reply:
A little verbosity is good here I believe